src/app/features/collectibles/components/collectible.layout.tsx

```diff
@@ -0,0 +1,65 @@
+import { ReactNode } from 'react';
+
+import { Box, Stack, Text } from '@stacks/ui';
+import type { BoxProps } from '@stacks/ui';
+import { color } from '@stacks/ui-utils';
+
+export interface CollectibleLayoutProps {
+  title: string;
+  subtitle: string;
+  backgroundElementProps?: BoxProps;
+  onClick?(): void;
+  children: ReactNode;
+}
+export function CollectibleLayout({
+  title,
+  subtitle,
+  onClick,
+  backgroundElementProps,
+  children,
+}: CollectibleLayoutProps) {
+  return (
+    <Box
+      _focus={{
+        outline: '4px solid #CEDAFA',
+      }}
+      as={onClick ? 'button' : 'div'}
+      borderRadius="16px"
+      onClick={onClick}
```

Comment:
nit picking but a custom `onSelectCollectable` event is nicer/more readable imo

Reply:
Agree here, missed changing it. Will do.